src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -3005,27 +3013,23 @@ ss::future<> tx_gateway_frontend::expire_old_tx(
     }
 
     auto term = term_opt.value();
+    auto timeout = config::shard_local_cfg().create_topic_timeout_ms();
 
-    co_await do_expire_old_tx(
-      stm,
-      term,
-      tx_id,
-      config::shard_local_cfg().create_topic_timeout_ms(),
-      false);
+    auto r0 = co_await get_tx(term, stm, tx_id, timeout);
+    if (!r0.has_value()) {
```

Comment:
do we want to log value here ?

Reply:
there is some logging already happening in get_tx; also expiring isn't triggered by an user action